sbncode/CAFMaker/FillReco.cxx

```diff
@@ -155,16 +156,28 @@ namespace caf
     srmatch.flashYWidth = match.flashYWidth;
     srmatch.flashZWidth = match.flashZWidth;
     srmatch.flashClassification = static_cast<int>(match.flashClassification);
-    for(const auto& matchedCRTHit : match.matchedCRTHits){
-      std::cout << "CRTPMTTimeDiff = "<< matchedCRTHit.PMTTimeDiff << "\n";
+    srmatch.matchedCRTHits.clear();
+    caf::SRMatchedCRT matchedCRT;
+    if(!match.matchedCRTHits.empty()){
+      matchedCRT.PMTTimeDiff = crtmatch.PMTTimeDiff; 
+      matchedCRT.time = crtmatch.time;
+      matchedCRT.sys = crtmatch.sys;
+      matchedCRT.region = crtmatch.region;
+      matchedCRT.position = SRVector3D(crtmatch.position.X(), crtmatch.position.Y(), crtmatch.position.Z());
+      std::cout << "CRTPMTTimeDiff = "<< matchedCRT.PMTTimeDiff << "\n";
+    }
+    std::cout << "CRTPMTTimeDiff = "<< matchedCRT.PMTTimeDiff << "\n";
+    srmatch.matchedCRTHits.push_back(matchedCRT);
+    /*for(const auto& matchedCRTHit : match.matchedCRTHits){
+     
       caf::SRMatchedCRT matchedCRT;
       matchedCRT.PMTTimeDiff = matchedCRTHit.PMTTimeDiff; 
       matchedCRT.time = matchedCRTHit.time;
       matchedCRT.sys = matchedCRTHit.sys;
       matchedCRT.region = matchedCRTHit.region;
       matchedCRT.position = SRVector3D(matchedCRTHit.position.X(), matchedCRTHit.position.Y(), matchedCRTHit.position.Z());
       srmatch.matchedCRTHits.push_back(matchedCRT);
-    }
+      }*/
```

Comment:
probably is better to remove outdated code and not comment out, unless it is expected to be re-addressed in the future.